src/Database.m

```diff
@@ -134,7 +134,7 @@ - (BOOL)initialiseDatabase {
     } else if (databaseVersion >= MA_Min_Supported_DB_Version) {
         NSAlert * alert = [[NSAlert alloc] init];
         [alert setMessageText:NSLocalizedString(@"Database Upgrade", nil)];
-        [alert setInformativeText:NSLocalizedString(@"Vienna must upgrade its database to the latest version. This may take a minute or so. We apologize for the inconveninece.", nil)];
+        [alert setInformativeText:NSLocalizedString(@"Database upgrade explanation text", @"Vienna must upgrade its database to the latest version. This may take a minute or so. We apologize for the inconvenience.")];
```

Comment:
I object to this change. We should strive for matching key-value pairs in the source files, so that we can get rid of the Localized.strings file in Base.lproj.